great_expectations/cli/v012/datasource.py

```diff
@@ -1227,7 +1230,7 @@ def _get_batch_kwargs_from_generator_or_from_file_path(
                             "\nDoes this file contain a header row?", default=True
                         )
                         batch_kwargs["reader_options"] = {"header": header_row}
-                    batch = datasource.get_batch(batch_kwargs=batch_kwargs)
+                    datasource.get_batch(batch_kwargs=batch_kwargs)
```

Comment:
Does this do something in-place?

Reply:
I doubt it.
I assume it's another implicit error check.

>if we can get a batch then there was no issue.

But it's worth flagging to @alexsherstinsky that the `batch` here are never used or returned.

Reply:
@Kilo59 If this is a `v012` code, I would leave it as is; we are not supposed to be maintaining it.  In fact, it is prime for full wholesale removal.  There have been a series of JIRA items, focused on removing the V2 codebase; perhaps we can either find and resurrect them or create fresh ones, and bring it up at a meeting.  Thanks!